vue-frontend/tailwind.config.cjs

```diff
@@ -13,7 +13,7 @@ module.exports = {
         xl: "1200px",
         xl2: "1330px",
         "2xl": "1400px",
-        "3xl": "3840px",
+        "3xl": "2440px",
```

Comment:
Why did the value of 3xl change?
parallax image looked good in full screen

Reply:
I think we haven't used 3xl anywhere in components